packages/web/src/pages/audio-rewards-page/ChallengeRewardsTile.tsx

```diff
@@ -139,15 +145,23 @@ const RewardPanel = ({
           />
         )}
       </div>
-      <ButtonWithArrow
-        className={wm(
-          cn(styles.panelButton, hasDisbursed ? styles.completed : '')
-        )}
-        completed={challenge?.state}
-        text={buttonMessage}
-        onClick={openRewardModal}
-        textClassName={styles.panelButtonText}
-      />
+      {hasDisbursed ? (
+        <Button
+          className={wm(cn(styles.panelButton, styles.completed))}
+          type={buttonType}
+          text={buttonMessage}
+          onClick={openRewardModal}
+          textClassName={styles.panelButtonText}
+        />
+      ) : (
+        <ButtonWithArrow
```

Comment:
at this point, maybe we just axe the `ButtonWithArrow` usage entirely and put the icon in above conditionally on the `hasDisbursed` flag? See convo in slack also